StepperEngine/src/Flow/Flow.java

```diff
@@ -71,12 +71,22 @@ private Flow() {
     }
 
     // Input validation to do:
-    // Non existent steps/data types
+    // Non-existent steps/data types
     // outputs with the same name
     private void setFlowLevelAliases(List<STFlowLevelAlias> aliases) {
         for(STFlowLevelAlias alias : aliases) {
             Step current = getStepByFinalName(alias.getStep());
-            current.trySetDataAlias(alias.getSourceDataName(), alias.getAlias());
+            if(current.trySetDataAlias(alias.getSourceDataName(), alias.getAlias()))throw new RuntimeException("In flow level aliasing:"+alias.getSourceDataName()+" does not exist");
```

Comment:
shouldn't this be if(!current.try....)  ?

Reply:
yes,true i missed that